packages/api-client/src/APIClient.ts

```diff
@@ -425,4 +425,23 @@ export class APIClient extends EventEmitter {
       }
     };
   }
+
+  /**
+   * Will check if MLS is supported by backend (whether used api version supports MLS and backend removal key is present).
+   */
+  public async isMLSSupportedByBackend(): Promise<boolean> {
```

Comment:
```suggestion
  public async supportsMLS(): Promise<boolean> {
```

We know we are dealing with the APIClient so no need for mentionning the backend at all in the name of the method. It's already explicit from the context